flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/enumerator/topic/range/FixedRangeGenerator.java

```diff
@@ -18,23 +18,40 @@
 
 package org.apache.flink.connector.pulsar.source.enumerator.topic.range;
 
+import org.apache.flink.annotation.PublicEvolving;
 import org.apache.flink.connector.pulsar.source.enumerator.topic.TopicMetadata;
 import org.apache.flink.connector.pulsar.source.enumerator.topic.TopicRange;
 
 import java.util.List;
 
+import static org.apache.flink.connector.pulsar.source.enumerator.topic.range.TopicRangeUtils.validateTopicRanges;
+
 /** Always return the same range set for all topics. */
+@PublicEvolving
 public class FixedRangeGenerator implements RangeGenerator {
```

Comment:
Unused or no tested?

Reply:
I add this only want to tell the end user. This class can be used.

Reply:
Then I tend to remove this class since it's untested and basically I don't think users can define `TopicRange` manually. If they want to do it, write their own implementation. Since it's untested, I don't since we will provide `PublicEvolving ` semantic.